flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/spec/DynamicTableSinkSpec.java

```diff
@@ -59,18 +58,14 @@ public class DynamicTableSinkSpec extends DynamicTableSpecBase {
     @JsonCreator
     public DynamicTableSinkSpec(
             @JsonProperty(FIELD_NAME_CATALOG_TABLE) ContextResolvedTable contextResolvedTable,
+            @Nullable @JsonProperty(FIELD_NAME_DYNAMIC_OPTIONS) Map<String, String> dynamicOptions,
             @Nullable @JsonProperty(FIELD_NAME_SINK_ABILITIES) List<SinkAbilitySpec> sinkAbilities,
             @Nullable @JsonProperty(FIELD_NAME_TARGET_COLUMNS) int[][] targetColumns) {
-        this.contextResolvedTable = contextResolvedTable;
+        super(contextResolvedTable, dynamicOptions);
         this.sinkAbilities = sinkAbilities;
         this.targetColumns = targetColumns;
     }
 
-    @JsonGetter(FIELD_NAME_CATALOG_TABLE)
```

Comment:
I am curious why are we removing this getter, when we still have @JsonGetter(FIELD_NAME_SINK_ABILITIES). 
Do non-nullable fields automatically have a getter?